### PR TITLE
Change server : nexus to sonatype

### PR DESCRIPTION
**Pull request by adrbessy**
versions of the dependencies had been updated

### CODE REVIEW

File: features/groovyeditor/pom.xml [hunk 1]
```diff
@@ -11,6 +11,6 @@
 		<relativePath>../</relativePath>
 	</parent>
 	<artifactId>groovyeditor</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
+	<!-- <version>1.0.0-SNAPSHOT</version> -->
```
Comment (SPalominos) [inline features/groovyeditor/pom.xml:14]: Please keep the version tag.

File: features/h2gis/pom.xml [hunk 2]
```diff
@@ -11,6 +11,6 @@
 		<relativePath>../</relativePath>
 	</parent>
 	<artifactId>h2gis</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
+	<!--  <version>1.0.0-SNAPSHOT</version> -->
```
Comment (SPalominos) [inline features/h2gis/pom.xml:14]: Please keep the version tag.

File: p2/pom.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
 	</parent>
 
 	<artifactId>p2</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
+	<!--  <version>1.0.0-SNAPSHOT</version>  -->
```
Comment (SPalominos) [inline p2/pom.xml:14]: Please keep the version tag.

**Review by SPalominos — CHANGES_REQUESTED** [review]

**Comment by ebocher** [comment]
This PR is merged with the second one

**Pull request closed without merging**